pom.xml

```diff
@@ -232,6 +233,41 @@
                         <goals>
                             <goal>report</goal>
                         </goals>
+                        <configuration>
+                            <haltOnFailure>false</haltOnFailure>
```

Comment:
@aman-sai Why are these changes required ?

Reply:
I've written it when the repo was using cobertura. Now it's using Jacoco. we don't need these changes now